core/src/main/java/org/commonjava/maven/ext/core/impl/DependencyManipulator.java

```diff
@@ -344,17 +344,23 @@ private void apply( final Project project, final Model model, final Map<Artifact
                                 }
                                 else
                                 {
-                                    logger.warn( "Parent reference {} replacement: {} of original version: {} violates the strict version-alignment rule!",
-                                                 ga( project.getModelParent() ), newValue, oldValue );
+                                    if ( logger.isWarnEnabled() )
```

Comment:
Probably redundant as we also have warn level enabled

Reply:
The issue with the `ga()` method call is a that it's calling `String.format()` inside a logger call, so we lose the point of using the `{}` placeholders.

A simple fix is to remove the guard and replace `String.format("%s:%s")` with `"logger.info("{}:{}", project.getGroupId(), project.getArtifactId())`.